techniques/systemSettings/userManagement/userManagement/7.1/metadata.xml

```diff
@@ -106,7 +107,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       </SELECT1>
       <INPUT>
         <NAME>USERGROUP_USER_SHELL</NAME>
-        <DESCRIPTION>Shell for this account</DESCRIPTION>
+        <DESCRIPTION>User shell</DESCRIPTION>
```

Comment:
You've removed "for this account" above, so in the same logic, this should just be "Shell"